docs/get-started/features.md

```diff
@@ -51,6 +51,9 @@ such as restricting discovered features with the -label-whitelist option.*
 | **`cpu-cstate.enabled`**          | bool   | Set to 'true' if cstates are set in the intel_idle driver, otherwise set to 'false'. Unset if intel_idle cpuidle driver is not active.
 | **`cpu-rdt.<rdt-flag>`**          | true   | [Intel RDT][intel-rdt] capability is supported. See [RDT flags](#intel-rdt-flags) for details.
 | **`cpu-sgx.enabled`**             | true   | Set to 'true' if Intel SGX is enabled in BIOS (based a non-zero sum value of SGX EPC section sizes).
+| **`cpu-model.vendor_id`**             | string   | Comparable CPU vendor ID.
+| **`cpu-model.family`**             | int   | CPU family model.
+| **`cpu-model.id`**             | int   | CPU model number.
```

Comment:
nit: fixing the alignment of the cells would make this more readable also in textfile form